Resources/Audio/Effects/Footsteps/attributions.yml

```diff
@@ -33,6 +33,14 @@
   copyright: "Taken and modified from tgstation (clownstep 1 and 2) by brainfood1183 (github)"
   source: "https://github.com/tgstation/tgstation/tree/f8ee37afc00bce1ad421615eaa0e4cbddd5eea90/sound/effects"
 
+- files:
+  - snake1.ogg
+  - snake2.ogg
+  - snake3.ogg
+  license: "Custom"
+  copyright: "Taken from https://zvukipro.com/"
+  source: "https://zvukipro.com/jivotnie/21-zvuki-zmej.html"
```

Comment:
Is there a license for these?

Reply:
Probably not, the author of the original audio authorised their free use. Judging by the author's comments at the bottom of the link.
I cut the sounds from the original heavily.